tests/common/utilities.py

```diff
@@ -1112,18 +1112,31 @@ def capture_and_check_packet_on_dut(
         duthost.file(path=pcap_save_path, state="absent")
 
 
-def duthost_ssh(duthost, sonic_username, sonic_passwords, sonic_ip):
-    for password in sonic_passwords:
+def paramiko_ssh(username, passwords, ipaddr):
```

Comment:
This would be a common function to be used by lots of other tests. Add docstring as a document is important. Can you add docstring for it?

Reply:
Name `ipaddr` may be conflicting with a library also named `ipaddr`.
The convention is to have ip address as the first argument. Then username and password.

Reply:
done